.appveyor.yml

```diff
@@ -43,6 +63,7 @@ install:
   - cmd: echo Using Python at %MESON_PYTHON_PATH%
   - cmd: if %compiler%==msvc2010 ( call "C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\vcvarsall.bat" %arch% )
   - cmd: if %compiler%==msvc2015 ( call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %arch% )
+  - cmd: if %compiler%==msvc2017 ( call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsDevCmd.bat" -arch=%arch% )
```

Comment:
Care to elaborate why? vcvarsall just calls VsDevCmd after parsing its arguments, so why don't we just call it directly?